pkg/skaffold/errors/errors.go

```diff
@@ -18,45 +18,34 @@ package errors
 
 import (
 	"errors"
-	"fmt"
 	"strings"
-	"sync"
 
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/constants"
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/instrumentation"
-	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/runner/runcontext"
 	"github.com/GoogleContainerTools/skaffold/proto/v1"
 	protoV2 "github.com/GoogleContainerTools/skaffold/proto/v2"
 )
 
 const (
 	// Report issue text
 	reportIssueText = "If above error is unexpected, please open an issue " + constants.GithubIssueLink + " to report this error"
```

Comment:
The links are quite big to fit in the middle of a sentence.
```suggestion
	reportIssueText = "If above error is unexpected, please open an issue to report this error at " " + constants.GithubIssueLink
```